LICENSE

```diff
@@ -1,202 +1,14 @@
+Copyright (c) 2016, The OpenThread Authors.
+All rights reserved.
 
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
```

Comment:
I think we really should be including our own copy of the license, not linking to a copy of it.